Assets/Content/Systems/Interactions/OpenInteraction.cs

```diff
@@ -20,7 +20,7 @@ public IClientInteraction CreateClient(InteractionEvent interactionEvent)
 
         public string GetName(InteractionEvent interactionEvent)
         {
-            return ((IGameObjectProvider)interactionEvent.Target).GameObject.GetComponent<Animator>().GetBool(OpenId) ? "Close" : "Open";
+            return ((IGameObjectProvider)interactionEvent.Target).GameObject.GetComponent<Animator>().GetBool(OpenId) ? "Open" : "Close";
```

Comment:
The previous line was correct. With this amendment, visibly closed lockers and toolbox will have a "Close" interaction. When clicked, they will visibly open. The next interaction will have "Open" and "View Container" as options. Please reverse this change.